src/Operations.jl

```diff
@@ -1855,9 +1855,9 @@ function sandbox(fn::Function, ctx::Context, target::PackageSpec, target_path::S
                 err isa Resolve.ResolverError || rethrow()
                 allow_reresolve || rethrow()
                 @debug err
-                @warn "Could not use exact versions of packages in manifest, re-resolving"
-                temp_ctx.env.manifest.deps = Dict(uuid => entry for (uuid, entry) in temp_ctx.env.manifest.deps if isfixed(entry))
-                Pkg.resolve(temp_ctx; io=devnull, skip_writing_project=true)
+                printpkgstyle(ctx.io, :Sandbox, "Could not use exact versions of packages in manifest. Re-resolving dependencies", color=Base.warn_color())
```

Comment:
Can we remove the `Sandbox` thing, I don't think that concept is ever exposed to users elsewhere.